src/runtime/vue/components/Link.vue

```diff
@@ -202,8 +176,8 @@ function resolveLinkClass({ route, isActive, isExactActive }: any = {}) {
 </script>
 
 <template>
-  <template v-if="hasRouter && !isExternal">
```

Comment:
Why remove this check? This could still be useful for using with `vue` without vue-router or other frameworks.

Reply:
I mean, we have this `vue-router` export in the stub: https://github.com/nuxt/ui/blob/v3/src/runtime/vue/stubs.ts#L10. It doesn't work without `vue-router` anyway.